libs/shared/contentful/src/lib/contentful.manager.ts

```diff
@@ -18,6 +19,8 @@ import {
   ServicesWithCapabilitiesResultUtil,
 } from './queries/services-with-capabilities';
 import { DeepNonNullable } from './types';
+import { PurchaseWithDetailsOfferingContentUtil } from './queries/purchase-with-details-offering-content/util';
+import { purchaseWithDetailsOfferingContentQuery } from './queries/purchase-with-details-offering-content';
```

Comment:
Nit: non-blockers 
- `PurchaseWithDetailsOfferingContentUtil` should be able to be imported from `./queries/purchase-with-details-offering-content` as well?
- order of imports (after L16)